run_codenarc.py

```diff
@@ -54,6 +55,11 @@ def _build_classpath(args):
     return ":".join(classpath)
 
 
+def _codenarc_version(version, is_groovy4):
+    """Get the CodeNarc version depending on the version of Groovy being used."""
+    return version if not is_groovy4 else f"Groovy4-${version}"
```

Comment:
```suggestion
    return f"Groovy4-${version}" if is_groovy4 else version
```
might read slightly better, but I don't feel strongly.